src/config/cambodia/layers.json

```diff
@@ -895,7 +895,7 @@
         "label": "Number of people affected"
       }
     },
-    "measure": "NumPeoAff",
+    "measure": "group_tc1fy38/NumPeoAff",
```

Comment:
@wadhwamatic the results get flattened so you need to use the measure name without the group name. We can also change that and decide to return the full metric name including the group name. What do you prefer?

Reply:
@ericboucher - see my earlier comment. Currently (after flattening), if I exclude the group name, the survey submission doesn't appear on the map. At a quick glance, it looks like Kobo doesn't allow a variable name to repeat across groups, so I think using the name without the group name would be simpler. We just need to make sure the value is available consistently. As of now, the measure value isn't showing up as valid. See the 'No Data' response and black point vs. the expected red.

Reply:
It loaded ok for me after the latest changes on legendToStops